src/app/static/src/app/store/translations/locales.ts

```diff
@@ -1474,6 +1476,7 @@ const pt: Partial<Translations> = {
     versionCountLabelPlural: "versões",
     versionCountLabelSingle: "versão",
     versions: "Versões",
+    VMMC: "Saídas de cobertura DMMPT2 VMMC",
```

Comment:
Should DMMPT and VMMC have different acronyms in fr/pt (like UNAIDS does) because of different word ordering?